sdks/python/apache_beam/typehints/native_type_compatibility.py

```diff
@@ -50,6 +50,17 @@ def _get_compatible_args(typ):
   return None
 
 
+def _get_args(typ):
```

Comment:
Should these changes be in a different PR?

Reply:
These are all functions that I needed over in typehints/schemas.py

Reply:
Ack.